blazor/message/customization.md

```diff
@@ -9,11 +9,11 @@ documentation: ug
 
 # Customization in Blazor Message
 
-The Message component allows you to customize the content display positions and appearance. This section explains the details about changing the content alignments and border styles for messages.
+The Message component allows user to customize the content display positions and appearance. This section explains the details about changing the content alignments and border styles for messages.
 
 ## Content Alignment
 
-Normally, the message content is aligned to the **left**. The Message component allows you to align the message content in the **center** or **right** through the `ContentAlignment` property.
+Normally, the message content is aligned to the **left**. The Message component allows user to align the message content in the **center** or **right** through the `ContentAlignment` property.
```

Comment:
use the below content
allows the user to

Reply:
Replaced the content.